CondFormats/SiPixelTransient/src/SiPixelTemplate.cc

```diff
@@ -1888,7 +1894,8 @@ bool SiPixelTemplate::interpolate(int id, float cotalpha, float cotbeta) {
   if (cotalpha < 0.f) {
     locBz = -locBx;
   }
-  return SiPixelTemplate::interpolate(id, cotalpha, cotbeta, locBz, locBx);
+  bool goodEdgeAlgo = false;
```

Comment:
I am not sure I follow the logic here.
Why is this variable redefine here instead of using the one passed as function argument?

Reply:
in any case why do you need a new variable?